File: cash.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int cents;
    do
    {
        cents = get_int("Enter your change: ");
    }
    while (cents <= 0);

    int coins = 0;

    while(cents >= 25)
        {
            cents = cents - 25;
            coins ++;
        }
    while(cents >= 10)
        {
            cents = cents - 10;
            coins++;
        }
    while(cents >= 5)
        {
            cents = cents - 5;
            coins++;
        }
    while(cents >= 1)
        {
            cents = cents - 1;
            coins++;
        }

        printf("You will need at least %i", coins);
}

File: cash.py
# TODO
from cs50 import get_float

count = 0

while True:
    owed = get_float("How much change is owed: ")
    if owed > 0:
        break
    cents = round(owed * 100)

while owed >= 25:
    owed = owed - 25
    count += 1

while owed >= 10:
    owed = owed - 10
    count += 1

while owed >= 5:
    owed = owed - 5
    count += 1

while owed >= 1:
    owed = owed - 1
    count += 1

print(count)